server/pulp/server/webservices/views/repositories.py

```diff
@@ -0,0 +1,1214 @@
+                                        schedule, failure_threshold, enabled)
+        display_call = scheduled_call.for_display()
+        display_call['_href'] = reverse(
+            'repo_sync_schedule_resource',
+            kwargs={'repo_id': repo_id, 'importer_id': importer_id,
+                    'schedule_id': scheduled_call['id']}
+        )
+        response = generate_json_response(display_call)
+        return generate_redirect_response(response, display_call['_href'])
+
+
+class RepoSyncScheduleResourceView(ScheduleResource):
+    """
+    View for a single scheduled repository sync.
+    """
+
+    def __init__(self):
+        """
+        Initialize RepoSyncScheduleResource.
+        """
+
+        super(RepoSyncScheduleResourceView, self).__init__()
+        self.manager = manager_factory.repo_sync_schedule_manager()
+
+    @auth_required(authorization.READ)
+    def get(self, request, repo_id, importer_id, schedule_id):
+        """
+        Retrieve information about a scheduled sync.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param importer_id: id of the importer
+        :type  importer_id: str
+        :param schedule_id: id of the requested scheduled repository sync
+        :type  schedule)id: str
+
+        :return: information about the requested scheduled sync
+        :rtype: django.http.HttpResponse
+        """
+
+        self.manager.validate_importer(repo_id, importer_id)
+        resource_href = reverse('repo_sync_schedule_resource',
+                                kwargs={'repo_id': repo_id, 'importer_id': importer_id,
+                                        'schedule_id': schedule_id})
+        return self._get(schedule_id, resource_href)
+
+    @auth_required(authorization.DELETE)
+    def delete(self, request, repo_id, importer_id, schedule_id):
+        """
+        Remove a scheduled repository sync from the importer.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param importer_id: remove the scheduled sync from this importer
+        :type  importer_id: str
+        :param schedule_id: id of the scheduled repository sync to delete
+        :type  schedule)id: str
+
+        :return: An empty response
+        :rtype: django.http.HttpResponse
+        :raises pulp_exceptions.MissingResource: if schedule_id does not exist for the importer
+        """
+
+        try:
+            self.manager.delete(repo_id, importer_id, schedule_id)
+        except pulp_exceptions.InvalidValue:
+            raise pulp_exceptions.MissingResource(schedule_id=schedule_id)
+        return generate_json_response(None)
+
+    @auth_required(authorization.UPDATE)
+    @json_body_required
```

Comment:
it is exactly the same case like with repo update - all params are optional, so if do not specify any of them you will do an update with actually updating anything 
https://github.com/pulp/pulp/pull/1742/files#diff-69de4359dc90d2ab726dbca11e4f32b4R254
i think the @json_body_allow_empty is right in this case, because if you put the @json_body_required it will contradict with the docs, which say that all params are optional and none of them are required.